finalPrep/modZeroHero.js

```diff
@@ -55,30 +74,60 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
-
+var scaryMonster = {
+  name: "Reggie",
+  smell: "smelly socks",
+  weight: 140,
+  citiesDestroyed: ["Tokyo, London, New York"],
+  luckyNumbers: [7,12,45],
+  address = {
+    number: 1234,
+    street: "Elm",
+    state: "Mississippi",
+    zip: 10001
+  },
```

Comment:
To give assign the key of `address` to a value which is an object data type, you should follow the same syntax as above which uses a colon instead of an equal sign: 
```
address: {
    number: 1234,
    street: "Elm",
    state: "Mississippi",
    zip: 10001
  },
```